ops/internal/manage/depsets.go

```diff
@@ -146,7 +141,8 @@ func (dc *DepsetChecker) checkOffchain(cfgs []DiskChainConfig) error {
 }
 
 // checkOnchain ensures that DisputeGameFactoryProxy and EthLockboxProxy addresses read from onchain
-// are the same for all chains in a depset
+// are the same for all chains in a depset. These checks are only performed for chains who have
+// already activated interop (i.e. interop_time < current unix timestamp).
```

Comment:
iiuc there is a period of time (I'm not sure for how long) where we would expect chains in a depset to share these proxy addresses _before_ the hf activation. 

I just want to check that we are happy to not be able to detect if the proxy addresses aren't in fact shared during that time?

It feels like this is not a check on the depset per se, but a check on some L1 contracts' property of chains in a given depset. I am wondering if it should be split out from the consistency check on the depset itself?

Reply:
This is the onchain param we can query [want](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L1/OptimismPortal2.sol#L126) to determine if `opcm.migrate` has been called for a specific chain.

Note that opcm.migrate may be called twice (may change later); when an already migrated 1-interop chain is being added to another interop set. But as soon as that boolean is set, you can be sure that the chain is using super roots.

So we should implement something like this for the onchain checks:
```
if optimismPortal.superRootsActive && len(depset) > 1 {
  // loop through depset and validate all members have same DisputeGameFactoryProxy and same EthLockboxProxy
}
```

Reply:
Added an issue to capture this work https://github.com/ethereum-optimism/superchain-registry/issues/1069